docs/users/cross-compilation.md

```diff
@@ -0,0 +1,144 @@
+---
+id: cross-compilation
+title: Cross compilation setups
+---
+
+Cross-building the same codebase with Scala 2.13 and Scala 3.3+ is common, but
+it collides with how Scalafix resolves rules and compiler options. A rule or
+`--scalac-options` flag that only works on Scala 2 can crash Scalafix when the
+Scala 3 compilation unit is processed, and vice versa. Until Scalafix grows
+conditional configuration, the safest way to avoid those failures is to split
```

Comment:
There is no plan to do conditional configuration so the "Until" can be misleading